chirp/src/chirp_server.c

```diff
@@ -253,7 +242,7 @@ static int update_all_catalogs(const char *url)
 
 	char *message = jx_print_string(j);
 
-	list_iterate(catalog_host_list, update_one_catalog, message);
+	list_iterate(catalog_host_list, (int (*) (void *, const void *)) catalog_query_send_update, message);
```

Comment:
Would be clearer to cast to `list_op_t` here.

(I wish C was smart enough to detect type compatibility in function-pointer arguments, but oh well.)